docs/source/faq.rst

```diff
@@ -0,0 +1,113 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page contains frequently asked questions about MOTECH with short answers to them.
+
+.. contents::
+    :depth: 2
+
+Can MOTECH run on Windows?
+--------------------------
+MOTECH is avaliable on Windows and it is possible to run the service on it, but it is not actively supported.
+
+Is MDS using any caching? How does that affect a cluster deployment?
+--------------------------------------------------------------------
+MDS is using datanucleus, which by default uses level1 and level2 cache. Sometimes it can cause data inconsistency
+across multiple server nodes in cluster deployment. To resolve this issue you can turn off level2 cache by setting
+``datanucleus.cache.level2.type`` property to ``none``. There may be slight performance drop per server in result of
+turning off level2 cache. Level1 cache should be kept as it works within one transaction, therefore not affecting
+cluster deployment.
+
+How to use SSL for database connections in MOTECH?
+--------------------------------------------------
+In order to establish secure SSL connection JDBC URL needs to be updated with JDBC driver property ``'useSSL'=true'``
+eg. if you are using mySQL at localhost JDBC URL should be ``jdbc:mysql://localhost:3306/?useSSL=true``. As the URL
+entered on the MOTECH bootstrap page is only the base for actual database connections urls, you should provide URL with
+SSL parameter to appropriate config files.
+
+For MDS it can be set by editing ``~/.motech/config/datanucleus.properties`` and changing the property ``javax.jdo.option.ConnectionURL``.
+For quartz you have to edit the variable ``org.quartz.dataSource.motechDS.URL`` in ``quartz.properties``. If your config
+source is FILE, then you have to edit the file in scheduler directory in ``~/.motech/config``. If your source is UI, you
+have to change the quartz configuration through the manage modules section in the Admin UI.
```

Comment:
Isn't it also required to install certificates and add JVM variables pointing to those certs?